### PR TITLE
Use nasm-rs for nasm version check

### PR DESCRIPTION
**Pull request by kornelski**
nasm-rs already has a version check.

### CODE REVIEW

File: build.rs [hunk 1]
```diff
@@ -126,14 +126,18 @@ fn build_nasm_files() {
     config_include_arg.push_str(&out_dir);
     config_include_arg.push('/');
     let mut nasm = nasm_rs::Build::new();
+    nasm.min_version(2,14,0);
```
Comment (barrbrain) [inline build.rs:129]: `cargo fmt` is failing here due to spacing.
```suggestion
    nasm.min_version(2, 14, 0);
```

**Review by barrbrain — APPROVED** [review]
Thank you, it appears fixes for all our workarounds have been merged upstream.

**Comment by barrbrain** [comment]
Unfortunately, this did away with the less confusing error message in the case of missing nasm. We will follow up on that.

**Comment by kornelski** [comment]
Fixed error message: https://github.com/medek/nasm-rs/pull/36

**Pull request merged**